common-arcana.lic

```diff
@@ -166,11 +166,6 @@ module DRCA
   end
 
   def release_cyclics
-    get_data('spells')
-      .spell_data
-      .select { |_name, properties| properties['cyclic'] }
-      .select { |name, _properties| DRSpells.active_spells.keys.include?(name) }
-      .map { |_name, properties| properties['abbrev'] }
-      .each { |abbrev| fput("release #{abbrev}") }
+    bput('release cyclic')
```

Comment:
That does seem to work.  I started a cyclic, then started my simple test method to use bput and spammed a bunch of RT inducing commands.  The cyclic released correctly once I was out of RT